proxy/stats_get_prediction.go

```diff
@@ -23,12 +23,16 @@ type StatsGetTask struct {
 }
 
 type StatsGetPrediction struct {
-	GGStriveAPIURL  string
-	predictionState PredictionState
-	statsGetTasks   map[string]*StatsGetTask
-	client          *http.Client
-	PredictReplay   bool
-	skipNext        bool
+	GGStriveAPIURL   string
+	predictionState  PredictionState
+	statsGetTasks    map[string]*StatsGetTask
+	client           *http.Client
+	PredictReplay    bool
+	skipNext         bool
+	cachedFollowReq  *http.Response
```

Comment:
This feels separate enough from predictstatsget to be a separate standalone feature.

Also the cached responses are probably better done as a Url->struct{response, body} map so this can be expanded easily without copy-pasting the cached response code.

Reply:
I created a new class for the cached responses.
There's also a new flag `unsafe-cache-follow` for the get_follow and get_block requests. Stats get prediction isn't needed for this now, but will still add these 2 requests to the cache if used.